### PR TITLE
first update

### PR DESCRIPTION
**Pull request by gitsonhub**
my first update

### CODE REVIEW

**Comment by gChafik** [comment]
Good catch!

File: sonams-code.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        <title>My new site</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="">
+    </head>
+    <body>
+        <!--[if lt IE 7]>
+            <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="#">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+        <h1>hi i'm sonam</h1>
```
Comment (gChafik) [inline sonams-code.html:18]: Nice to meet you Sonam!

**Review by gChafik — CHANGES_REQUESTED** [review]

File: sonams-code.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>My new site</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="">
+    </head>
+    <body>
+        <!--[if lt IE 7]>
```
Comment (cliftonbnoble) [inline sonams-code.html:15]: Looks Good

File: sonams-code.html (hunk 1 above)
Comment (gitsonhub) [reply sonams-code.html:18]: nice to meet you too Chafik

File: sonams-code.html (hunk 2 above)
Comment (gitsonhub) [reply sonams-code.html:15]: thanks Clifton

**Review by cliftonbnoble — APPROVED** [review]

**Review by gitsonhub — COMMENTED** [review]
good

**Review by gChafik — APPROVED** [review]

**Pull request merged**